src/app/wallet/send/send.component.html

```diff
@@ -73,7 +73,7 @@ <h5 fxFlex="16.6">To Account</h5>
               <md-radio-button value="anon_balance" disabled="{{ send.input === 'anon_balance' }}" fxFlex="16.6">Private</md-radio-button>
               <div fxFlex="50">
                 <span class="balance-label">Balance: </span>
-                <span class="balance-amount enable-select">{{ getBalance(send.input) }}</span>
+                <span class="balance-amount enable-select">{{ getBalance(send.output) }}</span>
```

Comment:
why change from input -> output type?

Reply:
the balance showing same before

Reply:
The balance will indeed show the same as before, but it's non-nonsensical as the input type defines the balance. Sure in the balance transfer tab they're both the same but still, doesn't make sense.

Reply:
if you see the modal name are different inside two cards. That the reason we need to pass the output rather then input

Reply:
This used to work, indeed it's supposed to be output.

Reply:
thanks